pmpro-approvals.php

```diff
@@ -1588,6 +1589,62 @@ public static function pmpro_email_filter( $email ) {
 
 	}
 
+	/**
+	 * Include user fields in the admin approval emails automatically.
+	 *
+	 * @since TBD
+	 * 
+	 * @param object $email The email object.
+	 */
+	public static function pmpro_include_user_fields_in_approvals_emails( $email ) {
+
+		// Bail if we don't have an $email object or the template is missing from the object.
+		if ( empty( $email ) || ! is_object( $email ) || ! isset( $email->template ) ) {
+			return;
+		}
+
+		// If the email is an admin email, include user fields for all Approval emails.
+		if ( ! in_array( $email->template, array( 'admin_approved', 'admin_denied', 'admin_notification_approval' ) ) ) {
+			return;
+		}
+
+		// Get the user_id from the email
+		$user = get_user_by( 'email', $email->data['user_email'] );
+		$user_id = $user ? $user->ID : 0;
+
+		if ( ! empty( $user_id ) ) {
+			//add to bottom of email
+			$field_groups = PMPro_Field_Group::get_all();
```

Comment:
This will fatal error when running PMPro versions before 3.4.

Quickest solution here is probably to add a `class_exists()` check a few lines up and `return;` if not. This can be a feature just for PMPro v3.4+